src/com/tekion/cricket/Match.java

```diff
@@ -0,0 +1,267 @@
+package com.tekion.cricket;
+
+import java.util.Scanner;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class Match {
+    private final Team team1;
+    private final Team team2;
+    enum Winner{
+        TEAM1,
+        TEAM2,
+        TIE,
+        STARTED,
+        DRAW,
+    }
+    private Winner winner;
+    private final int totalAvailableBalls;
+
+    public Match()
+    {
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter Number of Overs in the Match");
+        totalAvailableBalls = sc.nextInt()*6;
+        sc.nextLine();
+        String teamName;
+        System.out.println("Enter Name of the first Team");
+        teamName = sc.nextLine();
+        team1 = new Team(teamName, totalAvailableBalls);
+        System.out.println("Enter Name of the Second Team");
+        teamName = sc.nextLine();
+        team2 = new Team(teamName,totalAvailableBalls);
+        winner = Winner.STARTED;
+    }
+
+    int simulateToss()
+    {
+        int rand = ThreadLocalRandom.current().nextInt(0,100);
+        if(rand%2==0)
```

Comment:
can we use ternary operator here instead of `if-else` ?

Reply:
yes sir, that will be good. I'll do that.